checkov/main.py

```diff
@@ -517,6 +518,15 @@ def add_parser_args(parser: ArgumentParser) -> None:
                help='comma separated key:value string to filter policies based on Prisma Cloud policy metadata. '
                     'See https://prisma.pan.dev/api/cloud/cspm/policy#operation/get-policy-filters-and-options for '
                     'information on allowed filters. Format: policy.label=test,cloud.type=aws ', default=None)
+    parser.add('--secrets-scan-file-type',
+               default=[],
+               env_var='CKV_SECRETS_SCAN_TYPE',
```

Comment:
```suggestion
               env_var='CKV_SECRETS_SCAN_FILE_TYPE',
```
somehow `CKV_SECRETS_SCAN_TYPE` sounds misleading, like being able to adjust the scan type 😄 not file type